### PR TITLE
Managing Vehicles in the administrator menu

### PR DESCRIPTION
**Pull request by ShaunL12345**
STILL WIP 
updates the database initiation process to drop tables and creates data for the vehicle table
also adds a new screen for managing vehicles

### CODE REVIEW

File: HaulageApplication/Haulage/DatabaseSetup.cs [hunk 1]
```diff
@@ -20,97 +20,177 @@ public static string GetDatabasePath()
...
+        createTableScripts.Add(@" 
                 CREATE TABLE IF NOT EXISTS user (
                 UserID INTEGER PRIMARY KEY AUTOINCREMENT,
                 RoleID INTEGER NOT NULL,
                 FullName TEXT NOT NULL
                  );");
 
-                createTableScripts.Add(@"
+        createTableScripts.Add(@"
                 CREATE TABLE IF NOT EXISTS item (
```
Comment (40081168LJ) [inline HaulageApplication/Haulage/DatabaseSetup.cs]: Naming conventions should be followed throughout, used lowercase for table name, used starting acpital followed by camel case. Please update to "Item".

File: HaulageApplication/Haulage/DatabaseSetup.cs [hunk 2]
```diff
@@ -20,97 +20,177 @@ public static string GetDatabasePath()
...
+
+    private static void CreateTables(SQLiteConnection connection)
+    {
+
+        List<string> createTableScripts = new List<string>();
+
+        //Create User Table for Haulage Data;
+
+        createTableScripts.Add(@" 
                 CREATE TABLE IF NOT EXISTS user (
```
Comment (40081168LJ) [inline HaulageApplication/Haulage/DatabaseSetup.cs]: Naming conventions should be followed throughout, used lowercase for table name, used starting acpital followed by camel case. Please update to "User".

**Review by 40081168LJ — CHANGES_REQUESTED** [review]
Please see comments to be updated.

**Review by 40346200Aidan — DISMISSED** [review]
Looks good, Approved

File: HaulageApplication/Haulage/AdminPages/AddVehiclePage.xaml.cs
Comment (40081168LJ) [inline HaulageApplication/Haulage/AdminPages/AddVehiclePage.xaml.cs:1]: LJ - Update the clearing out function of SaveVehicle_Clicked, maybe display message on clear/fail/success, etc.

Comment (40081168LJ) [reply HaulageApplication/Haulage/AdminPages/AddVehiclePage.xaml.cs:1]: LJ - Also write to console for debugging.

File: HaulageApplication/Haulage/AdminPages/ManageVehiclesPage.xaml.cs
Comment (40081168LJ) [inline HaulageApplication/Haulage/AdminPages/ManageVehiclesPage.xaml.cs:1]: LJ - Update the clearing out function of RemoveVehicle_Clicked, maybe display message on clear/fail/success, etc.

File: HaulageApplication/Haulage/AppShell.xaml
Comment (40081168LJ) [inline HaulageApplication/Haulage/AppShell.xaml:1]: LJ - Remove this, as not needed.

File: HaulageApplication/Haulage/BaseClasses/Accounting/Vehicle.cs
Comment (40081168LJ) [inline HaulageApplication/Haulage/BaseClasses/Accounting/Vehicle.cs:1]: LJ - Update code for StatusTypes to be releavant.

File: HaulageApplication/Haulage/DatabaseExecutionServices/VehicleExecutionService.cs
Comment (40081168LJ) [inline HaulageApplication/Haulage/DatabaseExecutionServices/VehicleExecutionService.cs:1]: LJ - Clear spacing in SQL. Spacing between functions, clear to 1 line.

**Review by 40081168LJ — DISMISSED** [review]
LJ - Update sections and see if test coverage can at least get to 50% +.

File: HaulageApplication/Haulage/AdminPages/AddVehiclePage.xaml.cs
Comment (ShaunL12345) [reply HaulageApplication/Haulage/AdminPages/AddVehiclePage.xaml.cs:1]: have updated this

File: HaulageApplication/Haulage/AdminPages/ManageVehiclesPage.xaml.cs
Comment (ShaunL12345) [reply HaulageApplication/Haulage/AdminPages/ManageVehiclesPage.xaml.cs:1]: done

File: HaulageApplication/Haulage/AppShell.xaml
Comment (ShaunL12345) [reply HaulageApplication/Haulage/AppShell.xaml:1]: done

File: HaulageApplication/Haulage/BaseClasses/Accounting/Vehicle.cs
Comment (ShaunL12345) [reply HaulageApplication/Haulage/BaseClasses/Accounting/Vehicle.cs:1]: have updated this

File: HaulageApplication/Haulage/DatabaseExecutionServices/VehicleExecutionService.cs
Comment (ShaunL12345) [reply HaulageApplication/Haulage/DatabaseExecutionServices/VehicleExecutionService.cs:1]: have done this

**Review by SBoer33 — APPROVED** [review]

**Pull request merged**